libs/wingc/src/jsify/snapshots/base_class_with_lifted_field_object.snap

```diff
@@ -58,7 +58,7 @@ const $stdlib = require('@winglang/sdk');
 const $outdir = process.env.WING_SYNTH_DIR ?? ".";
 const std = $stdlib.std;
 const $wing_is_test = process.env.WING_IS_TEST === "true";
-const cloud = require('@winglang/sdk').cloud;
+const cloud = $stdlib.cloud;
```

Comment:
Should we merge those to:

```js
const { cloud, std } = $stdlib;
```

Reply:
I think it's doable but it will add some complexity, since right now the JSifier follows a simple model of generating one statement per AST statement. The change would also be tricky since currently the compiler allows `bring` statements anywhere in the AST, e.g.:

```js
if true {
  bring math;
}
```

In any case, sounds like a nice future optimization.